### ISSUE TITLE
$base-line-height non-px value causes grid image background-size problem

### ISSUE THREAD

**Issue by bgriffith** [ericam/susy#453]
Hey Eric,

If you set  a variable called $base-line-height to a value that isn't a px (for example 1.5) Susy grid won't output the background-size correctly for the image debug.

For example if the variable was given a value of 1.5, you'd get;

```css
background-size: 25%, 100% 1.5;
``` 

This causes the value to be ignored and result is that the grid column images don't appear correctly.

Apologies if this was done by design. Just a to note - We're not using Compass and it's vertical rhythm - this was by coincidence.

**Comment by grrowl** [comment]
+1 experiencing this ourselves. FWIW, it seems unitless line-height is interpreted as ems in this scenario ([MDN line-height](https://developer.mozilla.org/en-US/docs/Web/CSS/line-height#Examples))

**Comment by ericam** [comment]
You can turn off that integration by using `show-columns` rather than `show` for the debug setting. 

```scss
// either in the settings map:
$susy: (
  debug: (
    image: show-columns,
  ),
);

// or as a keyword:
@include container(show-columns);
```

**Comment by ericam** [comment]
This has been fixed by 433a0bad2da8b

**Comment by bgriffith** [comment]
That's great news. Thanks! :+1:
